### PR TITLE
Minor Update in Generating Test Spec

### PR DESCRIPTION
**Pull request by axkar**
This PR gives a minor update to the make test-spec target.

It also adds the recently forgotten minor update in dhcp_basic/topology.svg and dhcp_routes/Readme.adoc

## Checklist

Tick *relevant* boxes, this PR is-a or has-a:

- [ ] Bugfix
  - [ ] Regression tests
  - [ ] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [x] Test changes
  - [x] Checked in changed Readme.adoc (make test-spec)
  - [ ] Added new test to group Readme.adoc and yaml file
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [ ] Documentation content changes
  - [ ] ChangeLog updated (for major changes)
- [ ] Other (please describe):

### CODE REVIEW

**Review by troglobit — APPROVED** [review]
lgtm, but let's see what @mattiaswal has to say.

File: test/test.mk [hunk 1]
```diff
@@ -38,7 +38,8 @@ test-sh:
 	$(test-dir)/env $(base) $(mode) $(binaries) -i /bin/sh
 
 test-spec:
-	@sed 's/{REPLACE}/$(subst ",,$(INFIX_NAME)) $(GIT_VERSION)/'  $(spec-dir)/Readme.adoc.in > $(spec-dir)/Readme.adoc
+	@esc_infix_name="$(echo $(INFIX_NAME) | sed 's/\//\\\//g')"; \
```
Comment (mattiaswal) [inline test/test.mk:41]: Why?

**Review by mattiaswal — CHANGES_REQUESTED** [review]

File: test/test.mk (hunk 1 above)
Comment (axkar) [reply test/test.mk:41]: When INFIX_NAME contains '/' character, the `sed` command gets confused when '/' is used as a delimiter

**Comment by troglobit** [comment]
Sorry, you just got another merge conflict after I merged my branch.  Probably the dhcp routes test, so maybe you can just drop that commit of yours (I think I have the same change).

**Comment by axkar** [comment]
> Sorry, you just got another merge conflict after I merged my branch. Probably the dhcp routes test, so maybe you can just drop that commit of yours (I think I have the same change).

Yes, that was the problem. I think I'll still keep dhcp_basic/topology.svg from the commit, as it still gets generated.

**Review by mattiaswal — APPROVED** [review]

**Pull request merged**